commands/utils/quick-pick-image.ts

```diff
@@ -129,3 +130,13 @@ export async function quickPickDockerFileItem(actionContext: IActionContext, doc
     }
     return dockerFileItem;
 }
+
```

Comment:
I tried running 'Run ACR Task File' from command palette and after it asked for registry and os, I got "Cannot read property 'relativeFilePath' of undefined"

Reply:
Sorry. It was an error I made on the last commit with the output channel. I fixed it already when I set everything to use the docker output channel.